authors/apps/bookmarks/tests/test_bookmarks.py

```diff
@@ -0,0 +1,57 @@
+from django.urls import reverse
+from .base import BaseTestCase
+
+
+class BookmarkTestCase(BaseTestCase):
+    """
+    Class Test Case for bookmarks
+    """
+    def test_creates_bookmark(self):
+        url = '/api/articles/1/bookmarks/'
+        response = self.client.post(url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.data['bookmark']['article_title'], 'this is andela')
+        self.assertEqual(response.data['bookmark']['id'], 31)
+        self.assertEqual(response.data['bookmark']['article_id'], 1)
+        self.assertEqual(response.data['bookmark']['username'], 'adminuser')
+
+    def test_returns_error_if_user_attempts_to_bookmark_twice(self):
+        url = '/api/articles/1/bookmarks/'
+        self.client.post(url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        response = self.client.post(url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.status_code, 400)
+        self.assertIn(response.data['error'], "You already bookmarked this Article")
+
+    def test_gets_bootmarks(self):
```

Comment:
Please endeavor to name your functions appropriately and meaningfully.

Reply:
okay, let me fix this Asap. Thanks for the observation

Reply:
@mwinel I have fixed the issue